### ISSUE TITLE
Cannot tab through inline calendar

### ISSUE THREAD

**Issue by onesneakymofo** [Eonasdan/bootstrap-datetimepicker#1224]
Fiddle:  http://jsfiddle.net/090g1p2a/1/

Issue:
I'm working on accessibility for a website, and I cannot tab through the inline calendar.  Obviously, if they can't tab through the dates, their screen readers won't read them what dates they have selected.
